fuzz/src/utils/test_node_signer.rs

```diff
@@ -0,0 +1,64 @@
+// This file is Copyright its original authors, visible in version control
+// history.
+//
+// This file is licensed under the Apache License, Version 2.0 <LICENSE-APACHE
+// or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// You may not use this file except in accordance with one or both of these
+// licenses.
+
+use bitcoin::secp256k1::{PublicKey, SecretKey, Secp256k1};
+use bitcoin::secp256k1::ecdh::SharedSecret;
+
+use lightning::chain::keysinterface::{NodeSigner, Recipient};
+
+pub struct TestNodeSigner {
+	node_secret: SecretKey,
+}
+
+impl TestNodeSigner {
+	pub fn new(node_secret: SecretKey) -> Self {
+		Self { node_secret }
+	}
+}
+
+impl NodeSigner for TestNodeSigner {
```

Comment:
why do we have a duplicate implementation of TestNodeSigner? It's added in test_utils in the previous commit.

Reply:
reviewing, hang on